src/components/common/Buttons/BackButton.js

```diff
@@ -11,7 +11,9 @@ const StyledButton = styled(Button)`
   justify-content: center;
   align-content: center;
   margin: 10px auto;
-  margin-left: 10px;
+  position: absolute;
```

Comment:
I think we should test this in a real iPad first. @donavynhaley , and if we can avoid using pixels that probably would be the best.